utils/utils.mk

```diff
@@ -41,3 +41,6 @@ get_sequence_number = $(shell \
 # sub-makes:  the process of exporting a variable includes evaluating it, and
 # we don't want it evaluated except when it is explicitly $(CALL)'d.
 unexport get_sequence_number
+
+# Image registry for storing RPM build container images
+PB_CONTAINER_REG = $(shell cat ${HOME}/.config/pbench/ci_registry.name)
```

Comment:
I think Mr. Jenkins and I have come to the same conclusion:  this approach is fine iff the user happens to have this file; otherwise, there's a problem.  (That is, unlike the previous changes, there's no option here to pull the value from an environment variable...and I _think_ we want that, at least for the CI.)

Reply:
If you aren't sure what else to do, try
```
PB_CONTAINER_REG ?= $(shell cat ${HOME}/.config/pbench/ci_registry.name)
```
If `PB_CONTAINER_REG` is "not defined", then that will assign it, but it won't override an existing definition.

Now, whether having an environment variable by that name (as opposed to a `make` variable...) constitutes "defined" inside the `make` context is another question, one which is most easily resolved empirically....

(Note also that, the assignment may be reevaluated at "interesting" times, but, since the RHS is relatively cheap and has no side-effects, we shouldn't need to worry about that.)

Reply:
Thanks, I am trying this out in the latest rebase.  Hopefully this will work.